stats/v1/STATS_v1.0.0-Beta-1.yaml

```diff
@@ -0,0 +1,639 @@
+            - `CS` (Commercial Schedules)
+            - `AN` (Arrival Notice)
+            - `AN_NTF` (Arrival Notice Notifications)
+            - `DI` (Delivery Instruction)
+            - `DI_NTF` (Delivery Instruction Notifications)
+            - `RMC` (Reefer Monitoring Commercial)
+            - `RMC_NTF` (Reefer Monitoring Commercial Notifications)
+            - `RMO` (Operational Reefer Monitoring)
+            - `RMO_NTF` (Operational Reefer Monitoring Notifications)
+            - `IC` (IoT Commercial)
+            - `IC_NTF` (IoT Commercial Notifications)
+            - `IO` (IoT Operations)
+            - `IO_NTF` (IoT Operations Notifications)
+            - `DO` (Delivery Order)
+            - `DO_NTF` (Delivery Order Notifications)
+            - `TO` (Transport Order)
+            - `TO_NTF` (Transport Order Notifications)
+            - `CRO` (Container Release Order)
+            - `CRO_NTF` (Container Release Order Notifications)
+        method:
+          type: string
+          enum:
+            - POST
+            - GET
+            - PUT
+            - DELETE
+            - PATCH
+          example: GET
+          description: The HTTP Method used in the referenced API call.
+        path:
+          type: string
+          example: /v2/events
+          description: The context path of the reference API call.
+          x-stoplight:
+            id: 4fkn28q9gaqth
+        version:
+          type: string
+          x-stoplight:
+            id: s601oruo9y7da
+          example: 2.2.0
+          description: DCSA version of the API being used.
+        statusCode:
+          type: integer
+          x-stoplight:
+            id: i9k0l00sksmv5
+          format: int32
+          example: 200
+          description: "Http response status code of respective reference call.\r\nFor quality monitoring purposes.  "
+        params:
+          type: array
+          x-stoplight:
+            id: kwhdrf20ubqx3
+          description: The request query parameters being used.
+          minItems: 0
+          uniqueItems: true
+          example:
+            - eventTypes
+            - equipmentReference
+          items:
+            x-stoplight:
+              id: ns5q1jn8rh3jx
+            type: string
+        uniqueAdopterHash:
+          type: string
+          description: "SHA-256 of the adopter. \r\n\r\n> *Rules to be defined.*"
+          x-stoplight:
+            id: 9wfo73c55b9mb
+          example: ba7816bf8f01cfea414140de5dae2223b00361a396177a9cb410ff61f20015ad
+      required:
+        - domain
+        - method
+        - path
+        - version
+    StatSummary:
+      title: StatSummary
+      x-stoplight:
+        id: zksz5ad1zk3eo
+      type: object
+      properties:
+        date:
+          type: string
+          x-stoplight:
+            id: 5vou2ooc47d7y
+          format: date
+          example: '2023-11-01'
+          description: Statistics created date in `ISO 8601` format.
```

Comment:
Is it on purpose that you have this field inside the payload stats-summary array?